libs/utils/android/workloads/youtube.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, test_env):
     def run(self, exp_dir, video_url, video_duration_s, collect=''):
 
         # Initialize energy meter results
-        nrg_data, nrg_file = None, None
+        er = None
```

Comment:
I like this er short for (i guess) energy_deport... maybe it's more readable is we always use something like "nrg_report"?... if we want to go that way it requires an update all over the places where you are using "er" ;-)